### ISSUE TITLE
Possible small UI update suggestion

### ISSUE THREAD

**Issue by schalkneethling** [peterbe/mdn-nottranslated#2]
Hey @peterbe,

Wondering, should we perhaps update the suspect list UI to be as shown below? With that, we probably won't need the modal anymore either. Let me know, happy to implement :)

![Screenshot 2020-03-02 at 10 27 42](https://user-images.githubusercontent.com/10350960/75658174-7529fd00-5c70-11ea-98d8-7b5acb082b66.png)

**Comment by peterbe** [comment]
I'd rather you open to view it first because there are a lot of false positives (or whatever kids call it these days).

**Comment by schalkneethling** [comment]
> I'd rather you open to view it first because there are a lot of false positives (or whatever kids call it these days).

Fair enough. It makes sense to me. As I opened the issue I did consider that ;) The one thing I did not find intuitive was that the card was clickable. I reckon something as simple as adding `cursor: pointer;` will go some way toward making that more obvious.

**Comment by peterbe** [comment]
> > I'd rather you open to view it first because there are a lot of false positives (or whatever kids call it these days).
> 
> Fair enough. It makes sense to me. As I opened the issue I did consider that ;) The one thing I did not find intuitive was that the card was clickable. I reckon something as simple as adding `cursor: pointer;` will go some way toward making that more obvious.

Yes! Or something like this:

```
<a href={`#${suspect.slug}`} onClick={event => {
  doTheThingsToOpenModal()
}}>
```
